sale_advance_payment/views/account_payment.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+
+    <record id="view_account_payment_tree" model="ir.ui.view">
+        <field name="name">account.payment.tree</field>
+        <field name="model">account.payment</field>
+        <field name="mode">primary</field>
```

Comment:
Should priority="999999" be specified to apply all other inheritances first?

Reply:
There is no need.